plugin.video.vrt.nu/resources/lib/kodiwrapper.py

```diff
@@ -100,6 +100,13 @@ def has_socks():
     return has_socks.installed
 
 
+class SafeDict(dict):
```

Comment:
Why not use a standard `dict.get(key, fallback)`?

Reply:
This is being used when doing pyformat in localized formatted strings.

If an unknown `{foobar}` is encountered, we do not want it to be reason for failure, but instead show `{foobar}` in the resulting string.